docs/streams/developer-guide.html

```diff
@@ -893,13 +893,15 @@
             KeyValue&lt;String, Long&gt; next = range.next();
             System.out.println("count for " + next.key + ": " + value);
           }
+          range.close();
```

Comment:
@abbccdda I have not seen this PR being updated with a new commit yet. Are you on it?